setup.py

```diff
@@ -39,7 +39,7 @@
 
```

Comment:
One line above this one, we check for python version 3.6. Do we want to check for python version 3.7 instead?

Reply:
yes, thanks! will update now